### PR TITLE
Formatting, some clarification and fixes for peaks2genes intro tutorials

### PR DESCRIPTION
**Pull request by bebatut**

### CODE REVIEW

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md [hunk 1]
```diff
@@ -215,59 +238,84 @@ In order to convert the chromosome names we have therefore two things to do:
...
-You might have noticed that the UCSC file is in `BED` format and has a database associated to it. That's what we want for our peak file as well:
+You might have noticed that the UCSC file is in `BED` format and has a database associated to it. That's what we want for our peak file as well
 
 > ### {% icon hands_on %} Hands-on: Change format and database
 >
 > 1. Click on the **pencil icon** in the history entry of our peak region file:
 >      ![Pencil icon](../../images/edit_icon.png)
 > 2. Switch to the `Convert Format` tab
 > 3. Select `Convert Genomic Intervals To BED` and press **Convert**
+> 4. Edit the "Database/Build" to select "mm9", the database build for mice used in the paper
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: In my previous testing this was not necessary, which is why I removed a similar point in commit 29f1a970c87862c61b09c4c8170f09bd364adc25

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md [hunk 2]
```diff
@@ -215,59 +238,84 @@ In order to convert the chromosome names we have therefore two things to do:
...
+>
+> 4. **Replace Text** {% icon tool %}: Rerun this tool accordingly for chromosome Y
+> 5. Inspect the latest file through the **eye icon**
+>
+>    Have we been successful?
 >
 {: .hands_on}
 
-We have quite some files now and should take care that we don't loose track. Let's rename our latest result to something more handy, e.g. `Peak regions`.
+We have quite some files now and should take care that we don't loose track. Let's rename our latest result to something more handy, *e.g.* `Peak regions`.
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: Italic for `e.g.` and `i.e.` are usually not recommended, see https://english.stackexchange.com/questions/3911/correct-spelling-italicization-of-e-g-i-e

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md [hunk 3]
```diff
@@ -320,49 +369,66 @@ Since we have some nice data, let's draw a barchart out of it!
...
 > 2. Go to the history **Options menu** (gear symbol) and select the `Extract Workflow` option.
 >
 >    ![Extracting workflow in history menu](../../images/history_menu_extract_workflow.png)
 >
->    The center pane will change and you will be able to choose which steps to include/exclude and how to name the newly created workflow.
+>    The center panel will change and you will be able to choose which steps to include/exclude and how to name the newly created workflow.
+>
+> 3. **Uncheck** any steps that shouldn't be included in the workflow
+>
+>    Since we did some steps which where specific to our custom peak file, we might to exclude:
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: s/might to/might want to/

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md [hunk 4]
```diff
@@ -404,45 +473,55 @@ We need to generate a new BED file from the original peak file that contains the
 Now we cut out just the chromosome plus the start and end of the summit:
 
 > ### {% icon hands_on %} Hands-on: Cut out columns
-> 1. **Cut** {% icon tool %}: Run **Cut columns from a table** with the following settings:
+> 1. **Cut columns from a table** {% icon tool %} with the following settings:
 >   - **Cut columns**: `c1,c8,c9`
 >   - **From**: our latest history item
-> 2. The output from **Cut** will have the format `tabular`. Change it to `interval` since that's what the tool **Intersect** expects.
+> 
+>    The output from **Cut** will have the format `tabular`.
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: will be in `tabular` format.

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md [hunk 5]
```diff
@@ -404,45 +473,55 @@ We need to generate a new BED file from the original peak file that contains the
...
 
 We used our workflow to rerun our analysis with the peak summits. The **Group** tool again produced a list containing the amount of genes found in each chromosome.
-But woudln't it be more interesting to know about the amount of peaks in each unique gene? Let's rerun the workflow with different settings:
+But woudln't it be more interesting to know about the amount of peaks in each unique gene? Let's rerun the workflow with different settings!
 
 > ### {% icon hands_on %} Hands-on: Run a workflow with changed settings
-> 1. Open the workflow menu (top menu bar). Find the workflow you made in the previous section, and select the option **Run**
+> 1. Open the workflow menu (top menu bar)
+> 2. Find the workflow you made in the previous section, and select the option **Run**
 > 2. Choose as inputs our imported gene BED file and the result of the **Cut** tool
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: Update list numbering: `3.`, ...

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md (hunk 1 above)
Comment (bgruening) [reply topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: Not strictly needed, but there is a change that this helps people and in theory tools can check that the genome-build is the same ... e.g. intersect tools. We need a plan for the entire genome-build concept in Galaxy to go further. For the time being I would like to keep it and tell people that these thing exists.

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md (hunk 5 above)
Comment (bgruening) [reply topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: Done.

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md [hunk 6]
```diff
@@ -40,27 +50,36 @@ Let's start with a fresh history.
...
 
-Download the list of peak regions (the file `GSE37268_mof3.out.hpeak.txt.gz`) from
-[GEO](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37268) to your
-computer and upload it to Galaxy.
-
 > ### {% icon hands_on %} Hands-on: Data upload
 >
-> 1. Click on the upload button in the upper left ot the interface.
+> 1. Download the list of peak regions (the file [`GSE37268_mof3.out.hpeak.txt.gz`](https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE37268&format=file&file=GSE37268%5Fmof3%2Eout%2Ehpeak%2Etxt%2Egz)) from [GEO](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37268) to your computer 
+> 2. Click on the upload button in the upper left ot the interface
```
[the remark this answers is not in the document]
Comment (bgruening) [reply topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: The page is not that intuitive unfortunately. I think the ftp link works, the http not or something like that. If people are doing the tutorial without instructors its probably good to go the save way.

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md (hunk 1 above)
Comment (willdurand) [reply topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: Maybe you could write something like: `Make sure the database built for mice used in the paper "mm9" is selected in "Database/Build"`

Comment (bgruening) [reply topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: I'm fine to change the sentence, but @nsoranzo point was to remove it I think. @nsoranzo how strong do you feel about it? I can remove it.

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (bgruening) [reply topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: I added, in addition, a tipp for uploading via link.

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md (hunk 1 above)
Comment (nsoranzo) [reply topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: I actually agree with @willdurand's suggestion, it's basically what I have used in https://github.com/galaxyproject/training-material/blob/master/topics/introduction/tutorials/galaxy-intro-101/tutorial.md#-icon-hands_on--hands-on-ucsc-genome-browser

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (nsoranzo) [reply topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: Thanks @bgruening!

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md (hunk 1 above)
Comment (bgruening) [reply topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: I added a similar sentence.

Comment (nsoranzo) [reply topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: Thanks!

**Comment by bgruening** [comment]
:+1: from my site, even when the tests are failing. We can fix this later if it persists ....

**Comment by bebatut** [comment]
It is green!!!

**Pull request merged**